go.mod

```diff
@@ -161,7 +161,7 @@ require (
 	github.com/prometheus/common/sigv4 v0.1.0
 	github.com/prometheus/consul_exporter v0.8.0
 	github.com/prometheus/memcached_exporter v0.13.0
-	github.com/prometheus/mysqld_exporter v0.14.0
+	github.com/prometheus/mysqld_exporter v0.15.1
```

Comment:
Any new config options that need to be exposed since it's a minor version, it can have new features?